src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Location/ShowLocationHistory/readme.md

```diff
@@ -0,0 +1,41 @@
+# Show location history
+
+Display your location history on the map.
+
+![](ShowLocationHistory.jpg)
+
+## Use case
+
+You can track the user's location and display that on a map, for example to visualize how the user has moved through the world or to allow the user to re-trace their steps.
```

Comment:
A lot of "the user" in this sentence :). My wordsmithin' attempt is below:

"You can track device location history and display it as lines and points on the map. The history can be used to visualize how the user moved through the world, to retrace their steps, or to create new feature geometry. An unmapped trail, for example, could be added to the map using this technique."